CHANGELOG.md

```diff
@@ -9,6 +9,34 @@ Add new release notes in reverse numerical order (newest first) below this comme
 You may want to add temporary notes here for tracking as features are added, before a new release is ready.
 -->
 
+## v0.2.0
+
+This release adds the first set of community-contributed analyses to the repository.
+These modules are focused on cell type identification and annotation for specific ScPCA datasets.
+Note that many of these modules are still in development at this stage, and may not yet be fully functional.
+
+- `cell-type-dsrct`
+- `cell-type-ETO-ALL-03`
+- `cell-type-glioblastoma`
+- `cell-type-nonETP-ALL-03`
+- `cell-type-wilms-tumor-06`
+- `cell-type-wilms0tumor-14`
+
+
+This release also adds the following new modules developed by the Data Lab:
+
+- `hello-clusters`: a demonstration module for clustering analysis using the [`rOpenScPCA` package](https://github.com/AlexsLemonade/rOpenScPCA)
+- `seurat-conversion`: a module for converting `SingleCellExperiment` objects to Seurat objects, also using the `rOpenScPCA` package
+- `metacells`: a module that begins to explore the utility of metacell analysis within the ScPCA project
```

Comment:
This did already exist at the last release, but didn't have content yet https://github.com/AlexsLemonade/OpenScPCA-analysis/tree/v0.1.0/analyses/metacells

Reply:
For some reason, I thought it wasn't in the release notes, but it was...
```suggestion
```